internal/app/apid/main.go

```diff
@@ -215,3 +221,23 @@ func apidMain() error {
 
 	return errGroup.Wait()
 }
+
+func verifyExtKeyUsage(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
```

Comment:
`rawCerts` not used?

Reply:
yes, it's a callback from stdlib, so we have to have expected signature